packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -193,14 +193,12 @@ function prepareTextNode(node, key, text, createNodeId) {
   return textNode
 }
 
-function prepareStructuredTextNode(node, key, content, createNodeId) {
+function prepareRichTextNode(node, key, content, createNodeId) {
   const str = stringify(content)
-  const structuredTextNode = {
-    ...content,
```

Comment:
I would suggest to first deprecate this field first and name the other one ast. WDYT?
That's the beauty of GraphQL ❤️